code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -131,8 +131,8 @@
 	desc = "A medipen for surviving in the harshest of environments, heals and protects from environmental hazards. "
 	icon_state = "stimpen"
 	volume = 80
-	amount_per_transfer_from_this = 80
-	list_reagents = list("salbutamol" = 10, "coffee" = 20, "leporazine" = 20, "tricordrazine" = 15, "epinephrine" = 10, "omnizine" = 5)
+	amount_per_transfer_from_this = 83
+	list_reagents = list("nanites" = 3, "salbutamol" = 10, "coffee" = 20, "leporazine" = 20, "tricordrazine" = 15, "epinephrine" = 10, "omnizine" = 5, "stimulants" = 10)

```

Comment:
Techno. Mmm, considering the nanites, can Medical Mechs still scan and synthesize Nanites? Since I know that in the past they could(or atleast Nanites predecessor before Goofchem)